common/cli-utils.js

```diff
@@ -92,7 +104,14 @@ const addExtendedOptions = (program, options = {}) => {
     return program;
 };
 
+if (require.main === module) {
+    addBaseOptions(new Command());
```

Comment:
I don't want to add it by default, and let the script decide since they might want to customize it a bit

Reply:
this was mistakenly pushed. i added this for testing locally.